qmail-remote.c

```diff
@@ -2,6 +2,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
+#include <unistd.h>
```

Comment:
In general this change is of course correct and needed, and as usual I'm mildly worried only about the timing of the change in programs which many users are likely to still be heavily patching: this one and `smtpd`, mainly.

@schmonz gonna schmonz ;-) (and seriously, it's nice to be back here thinking about notqmail, so thank you) -- but I would _love_ to be able to feel that this sort of thing is getting worried about automatically. Then I could find new kinds of things to be mildly annoying about. @josuah, how can we help finish getting the patch-merge-checker into our regular autobuilds?

Reply:
As this a bit older: no, I don't think so. At least I can't provoke a warning on my system, most likely because (at least today, maybe not when these patches were originally done) these headers are in one or another indirect way automatically included. So for now I would call it "manual code audit", even if it was not true back then it would be true if I had to do them again.